docs/advanced_usage.rst

```diff
@@ -482,18 +483,35 @@ traces should be sent off. If an error occurs, a message will be displayed in
 the console, and changes can be made as needed.
 
 
+.. _uwsgi:
+
 uWSGI
 -----
 
-The default configuration of uWSGI applications does not include the
-``--enable-threads`` setting which must be set to ``true`` for the
-tracing library to run.  This is noted in their best practices doc_.
+The tracer and profiler support uWSGI when configured with the following:
+
+- Threads must be enabled with `enable-threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#enable-threads>`_ or with `threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#threads>`_ if running uWSGI in multithreaded mode.
+- If manual instrumentation and configuration is used, `lazy-apps <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`_ must be used.
```

Comment:
How can someone know if they meet this case?

Reply:
Which case? The first bullet or second? 

As for the second, I think we should standardize on what it means to use `ddtrace-run` and to not use it. In my usage here, "automatic instrumentation and configuration" equates to using `ddtrace-run` with environment variables though in fact that's not entirely the case because a user can still get that by simply adding `import ddtrace.bootstrap.sitecustomize`. Instead, the "manual" relies on code like  `patch()` or `patch_all()` and `tracer.configure()`.